src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -26,11 +26,25 @@ class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
     private $wasSkipped = array();
     private $isSkipped = array();
 
-    public function __construct(array $extraClockMockedNamespaces = array())
+    public function __construct(array $mockedNamespaces = array())
     {
-        if ($extraClockMockedNamespaces) {
-            foreach ($extraClockMockedNamespaces as $ns) {
-                ClockMock::register($ns.'\DummyClass');
+        foreach ($mockedNamespaces as $type => $namespaces) {
+            if (!is_array($namespaces)) {
+                $namespaces = array($namespaces);
+            }
+            if (is_int($type)) {
+                // BC with v2.8 to v3.0
+                $type = 'time-sensitive';
```

Comment:
triggering a deprecation is not possible, but I just added a simple "echo", ok for you?